psi4/driver/qcdb/molecule.py

```diff
@@ -1473,7 +1477,7 @@ def to_dict(self, force_c1=False, force_units=False, np_out=True):
             validated_molrec = qcel.molparse.from_arrays(speclabel=False, verbose=0, domain='qm', **molrec)
             forgive.append('fragment_charges')
             forgive.append('fragment_multiplicities')
-        compare_molrecs(validated_molrec, molrec, 6, 'to_dict', forgive=forgive, verbose=0)
+        compare_molrecs(validated_molrec, molrec, 'to_dict', atol=1.e-6, forgive=forgive, verbose=0)
```

Comment:
Are we still planning to define "global" tolerances for tests?

Reply:
When you use the qcel API (`fn(expected, computed, label=None, kwargs)`), then, yes, there's a global tol of 1.e-6. I actually really like the explicit choice that the psi4 API demands, so I'll keep adding them. Most all fns you can use either API but I figured molrecs was local enough to me to convert.